packages/vats/src/core/basic-behaviors.js

```diff
@@ -1,12 +1,53 @@
 // @ts-check
 import { E, Far } from '@endo/far';
-import { makeIssuerKit } from '@agoric/ertp';
+import { AssetKind, makeIssuerKit } from '@agoric/ertp';
 
+import { Nat } from '@agoric/nat';
 import { makeNameHubKit } from '../nameHub.js';
-import { BLD_ISSUER_ENTRY } from '../issuers.js';
 
 import { feeIssuerConfig, collectNameAdmins, makeNameAdmins } from './utils.js';
 
+// TODO/TECHDEBT: move to run-protocol?
+const Tokens = harden({
+  RUN: {
+    name: 'RUN',
+    denom: 'urun',
+    proposedName: 'Agoric RUN currency',
+    assetKind: AssetKind.NAT,
+    displayInfo: { decimalPlaces: 6 },
+  },
+  BLD: {
+    name: 'BLD',
+    denom: 'ubld',
+    proposedName: 'Agoric staking token',
+    assetKind: AssetKind.NAT,
+    displayInfo: { decimalPlaces: 6 },
+  },
+});
+
+/**
+ * In golang/cosmos/app/app.go, we define
+ * cosmosInitAction with type AG_COSMOS_INIT,
+ * with the following shape.
+ *
+ * The urun supplyCoins value is taken from geneis,
+ * thereby authorizing the minting an initial supply of RUN.
+ */
+// eslint-disable-next-line no-unused-vars
+const bootMsgEx = {
```

Comment:
It's used for its type later in the code. I could write the corresponding JSDoc type declaration, but I think it would lose something in translation (in particular, the example values).